### PR TITLE
#165274360: Adds basic theme layout

### PR DESCRIPTION
**Pull request by kisakyegordon**
#### What does this PR do ?
- Creates initial design for the home page.

#### Relevant pivotal tracker stories ?
[#165274360](https://www.pivotaltracker.com/story/show/165274360)

#### Screenshots ?
Screenshot 2019-04-16 at 21 00 04

### CODE REVIEW

File: src/components/filter/filter.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.checkbox{
+    margin: 0px;
+}
+
+.list-item{
+    display: flex;
+    justify-content: space-between;
+    width: 110px;
+}
```
Comment (dotNesh) [inline src/components/filter/filter.scss]: Leave a line at the end of this file.

File: src/components/footer/footer.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    display: flex;
+    justify-content: center;
+    color: #fff;
+    height: 95px;
+}
+.copyright{
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+}
```
Comment (dotNesh) [inline src/components/footer/footer.scss]: Leave a line at the end of this and every other file.

**Comment by dotNesh** [comment]
@kisakyegordon what do you think about using `.jsx` instead of `.js` for your component files?

File: src/components/footer/footer.scss (hunk 2 above)
Comment (kisakyegordon) [reply src/components/footer/footer.scss]: i have modfied this

**Review by ayz4sci — APPROVED** [review]
LGTM

**Pull request merged**